certified-connectors/Oodrive Sign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2891 @@
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "x-ms-summary": "Skip first n contracts",
+            "description": "The sequence number of contract that will be skipped.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "get_properties",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "x-ms-summary": "Show contract properties",
+            "description": "If true, contract properties will be listed in the response.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "get_perimeters",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "x-ms-summary": "Show contract perimeters",
+            "description": "If true, contract perimeters will be listed in the response.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "before",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "x-ms-summary": "Created before",
+            "description": "Will list contract created before the given date.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "after",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Will list contract created after the given date.",
+            "x-ms-summary": "Created after",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "perimeter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Only with perimeter",
+            "description": "List contracts only in a given perimeter.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "status",
+            "in": "query",
+            "x-ms-summary": "Only with status",
+            "type": "array",
+            "collectionFormat": "csv",
+            "description": "Retrieve contract only with specified status",
+            "x-ms-visibility": "advanced",
+            "items": {
+              "type": "string",
+              "enum": [
+                "OPEN",
+                "CLOSED",
+                "SIGNED",
+                "PENDING",
+                "ARCHIVED",
+                "ABANDONED"
+              ]
+            }
+          }
+        ],
+        "x-ms-visibility": "important"
+      }
+    },
+    "/contracts/{id}/transaction/": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "$ref": "#/definitions/Transaction"
+            }
+          }
+        },
+        "summary": "Get a contract's status",
+        "description": "Check a specific contract's status",
```

Comment:
@QwerioTEAM all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)
Please fix this for all the operations in the swagger file